src/activity.rs

```diff
@@ -106,11 +106,11 @@ struct RsqrtOutput {
 }
 
 /// Fixed point `rsqrt` for `ssim_boost`
-fn ssim_boost_rsqrt(x: u64) -> RsqrtOutput {
+const fn ssim_boost_rsqrt(x: u64) -> RsqrtOutput {
   const INSHIFT: u8 = 16;
   const OUTSHIFT: u8 = 14;
 
-  let k = ((ILog::ilog(x) - 1) >> 1) as i16;
```

Comment:
Just making sure, is the removal of the `- 1` intentional here? My guess is yes given the docs in v_frame: `floor(log2(self)) + 1, or 0 if self == 0`, but want to be certain this is bit-identical to the prior output.

Reply:
Yes, it's intentional and I did make sure they're equivalent for x != 0: https://rust.godbolt.org/z/Gh5TxbEYK